idaes/models_extra/gas_solid_contactors/unit_models/tests/test_FB0D.py

```diff
@@ -233,13 +233,9 @@ def iron_oc_unscaled(self):
     @pytest.mark.skipif(solver is None, reason="Solver not available")
     @pytest.mark.component
```

Comment:
This looks like it should be an integration test - it looks like you are solving a (simple) dynamic problem, which is generally more than we put in `component` tests. Note that component tests should take no longer than 10 seconds to run (with very few exceptions).